src/components/Navigation/Pages/PageFiles/PageFiles.js

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation and others. Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+import React, { Component } from 'react'
+import { Grid, Row, Col } from 'react-bootstrap'
+import Attachments from '../../../../utils/attachments'
+
+export default class PageFiles extends Component {
```

Comment:
Metaquestion. Should this be `PageFile` (singular). The page will show the content of just one file right?

That should be reflected in the code and the file/directory naming.